pkg/artifactory/resource/webhook/resource_artifactory_webhook_base.go

```diff
@@ -82,3 +94,68 @@ var baseWebhookBaseSchema = func(webhookType string) map[string]*schema.Schema {
 		},
 	}
 }
+
+
+var baseWebhookBaseSchemaV2 = func(webhookType string) map[string]*schema.Schema {
```

Comment:
since these are not in a closure scope (and, I assume that's what you want) then don't use `var` - declare a normal function